src/pages/messaging/agent-chat-api/v3.4/index.mdx

```diff
@@ -1360,7 +1363,12 @@ https://api.livechatinc.com/v3.4/agent/action/unfollow_chat \
 
 ### Transfer Chat
 
-Transfers a chat to an agent or a group. The transferred chat must be active. When transferring directly to an agent, the agent must have access to the transferred chat.
+Transfers a chat to an agent or a group. The following restrictions apply:
+
+- The transferred chat must be active.
+- The requester must be present on the list of chat users.
+  - You can override it by setting the `ignore_requester_presence` parameter.
```

Comment:
Is the space at the beginning on purpose?  ;) If so, perhaps using `;` would be a better solution here?

Reply:
it's on purpose, this is how you create sub-lists; and it's better that way